torch/csrc/jit/codegen/cuda/kernel_ir.h

```diff
@@ -225,12 +226,6 @@ class TORCH_CUDA_CU_API Allocate final : public Expr {
     return alias_;
   }
 
-  void setAlias(const Allocate* alias) {
```

Comment:
No longer allowed to change the state. Alias pointer must be provided as part of a constructor call